setup.py

```diff
@@ -35,7 +35,7 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        "munkres==1.1.2",
+        "munkres>=1.0.6",
```

Comment:
are these version commits related to this patch?

Reply:
They're not, but `tox -e check` was failing without it.

Reply:
I'll try reverting it and see if travis ci passes.

Reply:
CI fails if I revert.